appinfo/info.xml

```diff
@@ -37,7 +37,7 @@ The First run wizard can be customized to meet specific design goals, or to chan
 		<navigation>
 			<id>firstrunwizard_about</id>
 			<name>About</name>
-			<route>files.view.index</route>
+			<route />
```

Comment:
This looks wrong?! Now the settings menu entry does not open for me anymore.

Reply:
lint info.,xml agrees:
```
 ./appinfo/info.xml:40: element route: Schemas validity error : Element 'route': [facet 'minLength'] The value has a length of '0'; this underruns the allowed minimum length of '1'.
./appinfo/info.xml:40: element route: Schemas validity error : Element 'route': [facet 'pattern'] The value '' is not accepted by the pattern '[0-9a-zA-Z_]+(\.[0-9a-zA-Z_]+){2}'.
./appinfo/info.xml:40: element route: Schemas validity error : Element 'route': '' is not a valid value of the atomic type 'route'.
./appinfo/info.xml fails to validate
```

Reply:
Fixed

This is only meant to work on top of https://github.com/nextcloud/firstrunwizard/pull/815 and https://github.com/nextcloud/server/pull/36232

Reply:
`xmllint` schema updated in https://github.com/nextcloud/server/pull/36449

Reply:
`xmllint` schema updated to allow no route in https://github.com/nextcloud/server/pull/36508